cuenca/resources/base.py

```diff
@@ -71,8 +81,8 @@ class Queryable(Resource):
 
     @classmethod
     def one(cls, **query_params) -> Resource:
-        q = cls._query_params(limit=2, **query_params)
-        resp = session.get(cls._resource, q.dict())
+        query = cls._query_params(limit=2, **query_params)
+        resp = session.get(cls._resource, query.dict(exclude_none=True))
```

Comment:
@felipaoo Why did you change this, and why is it inconsistent with `.all()`, `.first()` and `.count()`?

Reply:
If the idea is to exclude `None` to all queries, you should change the bas3 `QueryParams` in cuenca-validations to something like this:

```python
from pydantic import BaseModel

class QueryParams(BaseModel):
    def dict(self, exclude_none=True, **kwargs):
        return super().dict(exclude_none=exclude_none, **kwargs)
```

Reply:
in fact ... it's already there! https://github.com/cuenca-mx/cuenca-validations/blob/main/cuenca_validations/types/queries.py#L28-L33

Please remove your change. It's entirely redundant.

Reply:
removed